src/System.Runtime.Analyzers/Core/SystemRuntimeAnalyzersResources.resx

```diff
@@ -316,10 +316,10 @@
     <value>A method or constructor calls one or more members that have overloads that accept a System.IFormatProvider parameter, and the method or constructor does not call the overload that takes the IFormatProvider parameter. When a System.Globalization.CultureInfo or IFormatProvider object is not supplied, the default value that is supplied by the overloaded member might not have the effect that you want in all locales. If the result will be based on the input from/output displayed to the user, specify 'CultureInfo.CurrentCulture' as the 'IFormatProvider'. Otherwise, if the result will be stored and accessed by software, such as when it is loaded from disk/database and when it is persisted to disk/database, specify 'CultureInfo.InvariantCulture'</value>
   </data>
   <data name="SpecifyIFormatProviderMessageIFormatProviderAlternateString" xml:space="preserve">
-    <value>Because the behavior of '{0}' could vary based on the current user's locale settings, replace this call in '{1}' with a call to '{2}'.</value>
+    <value>The behavior of '{0}' could vary based on the current user's locale settings, replace this call in '{1}' with a call to '{2}'.</value>
   </data>
   <data name="SpecifyIFormatProviderMessageIFormatProviderAlternate" xml:space="preserve">
-    <value>Because the behavior of '{0}' could vary based on the current user's locale settings, replace this call in '{1}' with a call to '{2}'.</value>
+    <value>The behavior of '{0}' could vary based on the current user's locale settings, replace this call in '{1}' with a call to '{2}'.</value>
```

Comment:
I have moved the suggestion part of the diagnostic message over to the descriptor. This will help us in keeping the message short and also not lose any of the vital information that we want to pass on to the user